object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+/* exported toObject */
+function toObject(keyValuePair) {
+  const [key, value] = keyValuePair;
```

Comment:
This is a shorthand. But longhand would've also been fine:
const key = keyValuePair[0];
const value = keyValuePair[1];